ephypype/commands/tests/test_cli.py

```diff
@@ -4,10 +4,13 @@
 #
 # License: BSD (3-clause)
 
-import os
-import os.path as op
-from ephypype.commands import neuropycon
-from click.testing import CliRunner
+import matplotlib
+matplotlib.use('Agg')  # for testing don't use X server
+
+import os # noqa
+import os.path as op # noqa
+from ephypype.commands import neuropycon # noqa
+from click.testing import CliRunner # noqa
```

Comment:
I think you just need to do it for the first import. Better not to use it everywhere ...

Reply:
Yepp, you're right. Fixed it